new-client/src/plugins/DocumentHandler/DocumentHandler.js

```diff
@@ -9,7 +9,6 @@ import Observer from "react-event-observer";
 import MapViewModel from "./MapViewModel";
 import { withTheme, createMuiTheme } from "@material-ui/core/styles";
 import { deepMerge } from "../../utils/DeepMerge";
-import { hfetch } from "utils/FetchWrapper";
```

Comment:
@jesade-vbg has converted all files that were in `develop` at a given time to make use of `hFetch`. Please don't revert his work now.

Reply:
Yes, but i have since then moved the logic from the file DocumentHandler to DocumentHandlerModel so the same functionality is now present in another DocumentHandlerModel.js instead (with hfetch)

Reply:
Excellent